UnityProjects/MRTKDevTemplate/Assets/BuildAssets/BuildApp.cs

```diff
@@ -47,6 +48,28 @@ public static void StartCommandLineBuild()
             EditorApplication.Exit(success ? 0 : 1);
         }
 
+        public static void EnsureTMPro()
```

Comment:
Are we able to call the method in RuntimeTestUtilities from the yaml instead of adding a second one here? Or are they distinct enough that we want them both?

Reply:
Deleted the one in `RuntimeTestUtilities`.

Reply:
Deleting the dupe.